package.json

```diff
@@ -30,7 +30,14 @@
   },
   "homepage": "http://material-ui.com/",
   "dependencies": {
-    "react-draggable2": "^0.5.1"
+    "react": ">=0.14.0-rc1",
+    "react-dom": ">=0.14.0-rc1",
```

Comment:
Yes I agree, this change is fine. But what about the docs/package.json? It seems that this will only install already installed dependencies.